cli/src/main/java/org/jboss/sbomer/cli/feature/sbom/command/auto/GenerateCommand.java

```diff
@@ -90,7 +99,20 @@ public Integer call() throws Exception {
         }
 
         // It is able to read both: JSON and YAML config files
-        Config config = objectMapper.readValue(configPath.toAbsolutePath().toFile(), Config.class);
+        Config config;
+
+        try {
+            config = objectMapper.readValue(configPath.toAbsolutePath().toFile(), Config.class);
+        } catch (StreamReadException e) {
+            log.error("Unable to parse the configuration file", e);
+            return 2;
```

Comment:
Wouldn't be bad to have the statuses mapped internally with constants maybe? Just for improved visibility.

Reply:
Yes, I think it makes sense!

I was also thinking about storing the exit code in the database as well so that we could query based on the status. If we would add on top of this some kind of enum, then we would have even a nice failure reason in the entity, a description in text format and this would be in a single place.

Reply:
Yes good idea! Anything which will make our life easier to query and debug is surely welcome, talking for experience with PNC which has so many use cases. Go for it! :-)